webui/src/pages/repositories/repository/fileRenderers/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import remarkGfm from 'remark-gfm'
+import remarkHtml from 'remark-html'
+import ReactMarkdown from 'react-markdown';
+import { IpynbRenderer as NbRenderer } from "react-ipynb-renderer";
+
+export const MarkdownRenderer = ({ content }) => (
+    <ReactMarkdown remarkPlugins={[remarkGfm, remarkHtml]} linkTarget={"_blank"}>
```

Comment:
Note that in `repositories/repository/objects.jsx`, we are also presenting a markdown - consider replacing it with your new component.